assets/schema_input.json

```
@@ -10,15 +10,40 @@
             "sample": {
                 "type": "string",
                 "pattern": "^\\S+$",
-                "errorMessage": "Sample name must be provided and cannot contain spaces"
+                "errorMessage": "Sample name must be provided and cannot contain spaces",
+                "meta": ["id"]
             },
-            "fastq_1": {
+            "ontreads": {
                 "type": "string",
+                "format": "file-path",
                 "pattern": "^\\S+\\.f(ast)?q\\.gz$",
-                "errorMessage": "FastQ file for reads 1 must be provided, cannot contain spaces and must have extension '.fq.gz' or '.fastq.gz'"
+                "errorMessage": "ONT reads must be provided, cannot contain spaces and must have extension '.fq.gz' or '.fastq.gz'"
```

From the `must be provided` sentence I understand that these parameters are required, but they are not provided in the `required` field, so I would remove this bit from all the error messages.
```suggestion
                "errorMessage": "ONT reads cannot contain spaces and must have extension '.fq.gz' or '.fastq.gz'"
```

---

Good point, I have updated this file, see #93